+++
url = "research.html"

+++

<p>
<img src="images/blank.jpg">
<img src="images/research.jpg">
<p>


<table>
<tr>
<td width=20%></td>
<td width=60%>

<strong>Chemical Strategies for the Global Analysis of Enzyme Function</strong><br>

Our research group aims to understand the roles that mammalian enzymes play in physiological and pathological processes and to use this knowledge to identify novel therapeutic targets for the treatment of human disease. To achieve these goals, we develop and apply new technologies that bridge the fields of chemistry and biology, ascribing to the philosophy that the most significant biomedical problems require creative multidisciplinary approaches for their solution. Our technological innovations address fundamental challenges in systems biology that are beyond the scope of contemporary methods. For instance, enzymes are tightly regulated by post-translational events in vivo [1], meaning that their activity may not correlate with expression as measured by standard genomic and proteomic approaches. Considering that it is an enzyme's activity, rather than abundance that ultimately dictates its role in cell physiology and pathology, we have introduced a set of proteomic technologies that directly measures this parameter. These activity-based protein profiling (ABPP) methods [2,3] exploit the power of chemistry to engender new tools and assays for the global analysis of enzyme activities. The enzyme activity profiles generated by ABPP constitute unique molecular portraits of cells and tissues that illuminate how metabolic and signaling networks are regulated in vivo. Additionally, by evaluating enzymes based on functional properties rather than mere abundance, ABPP acquires high-content proteomic information that is enriched in novel markers and targets for the diagnosis and treatment of human disease [4-6].
<p>

We complement these efforts in technology development with focused studies on individual enzymes. With particular interests in the nervous system and cancer, we select proteins, such as the endocannabinoid-degrading enzyme fatty acid amide hydrolase (FAAH) [7], for detailed investigation using a range of chemical, biochemical, genetic, and pharmacological techniques. This multidisciplinary approach ensures that we generate all of the tools and models required to assign molecular, cellular, and physiological functions to enzymes and, as an important corollary, assess their suitability as therapeutic targets. Notably, these basic discovery projects both benefit from and provide a fertile testing ground for our technological innovations. Thus, through the integration of two complementary research programs (Figure 1), one dedicated to methods development for functional proteomics, and the other to the characterization of key signaling enzymes, our group achieves a unique balance that cultivates the creation and rapid implementation of cutting-edge technologies that address unmet needs in experimental biology.
<p>
<center><img src="images/overview.jpg"></center>
<p>
<em>Figure 1. Cravatt Group Research Programs - The Integration of Technology Development and Basic Discovery. We aim to create an integrated platform for the discovery, functional characterization, and therapeutic evaluation of any mammalian enzyme. Examples of enzymes, substrates, and inhibitors discovered by our research programs are highlighted throughout this Research Statement. 
</em>
<p>
<center><hr width=85%></center>
<p> 
<a name="abpp"></a>
<strong>Technology Development: Activity-Based Protein Profiling (ABPP)</strong><br>
In the post-genome era, researchers are charged with the task of assigning functions to thousands of newly predicted gene products. Proteomics aims to accelerate this process by developing methods for the parallel analysis of large numbers of proteins [8]. However, most proteomic technologies measure protein abundance [9,10] and, therefore, like genomics, provide only an indirect estimate of protein activity [1]. We believe that this critical shortcoming limits the information content achievable in standard genomic and proteomic experiments, thus impeding the characterization of key proteins that function as fundamental contributors to human disease. To address this central problem, we have introduced a chemical strategy referred to as activity-based protein profiling (ABPP) that utilizes active site-directed probes to profile the functional state of enzymes in whole proteomes [2,3] (Figure 2). ABPP probes label active enzymes, but not their inactive precursor or inhibitor-bound forms [4,12], and thus report on the major post-translational events that regulate enzyme function in vivo [1]. To date, we have developed ABPP probes for several enzyme classes, including serine hydrolases [11,12], metalloproteases [13], oxidoreductases [14,15], and glutathione S-transferases [14]. Examples of our current and future research efforts to apply and advance the ABPP technology are summarized below.
<p>
<center><img src="images/abpp.jpg"></center>
<p>
<em>Figure 2. Activity-based protein profiling (ABPP). ABPP probes label active, but not inactive (e.g., inhibitor-bound, zymogen) enzymes in complex proteomes [12]. Labeled enzymes are detected by in-gel fluorescence scanning [4] and identified by affinity purification and MS analysis [12].
</em>
<p> 
<center><hr width=85%></center>
<p>

<strong>Biological applications of ABPP - profiling enzyme activities in human cancer.</strong><br>
Despite intense basic and clinical research, annual deaths due to cancer remain relatively constant and cancer diagnoses continue to increase [16]. One of the major problems facing the eradication of cancer is a dearth of molecular markers and targets for the early diagnosis and specific treatment of this disease. Although DNA microarrays have provided transcriptional profiles that classify tumors into sub-groups predictive of clinical outcome [17] and metastatic potential [18], these genomic studies have yet to identify a conserved set of cancer-associated gene products. Indeed, if anything, these studies have emphasized the remarkable molecular heterogeneity of cancer, leading some to question whether general strategies for the diagnosis and treatment of this disease are even possible [16]. We hypothesize that general markers and targets for the diagnosis and treatment of cancer do exist, but that global technologies to measure proteins, and protein activity in particular, are required for their discovery and validation. To test this premise, we are applying ABPP to an array of in vitro and in vivo human cancer models, as well as primary tumor specimens.
<p>
Using ABPP, we have identified a group of enzyme activities that are upregulated in invasive human cancer cell lines from several different tumor types [4]; these enzymes include proteases, like urokinase, with recognized roles in tumorigenesis, and novel enzymes, such as the integral membrane hydrolase KIAA1363, for which no previous link to cancer had been made. More recently, we have found that human breast cancer lines, following growth in vivo as xenograft tumors, exhibit profound post-transcriptional changes in the activity of specific proteases and metabolic enzymes that correlate with increased rates of tumor growth and metastasis [5]. Together, these findings suggest that a common set of enzymes may support the progression of tumors from a variety of origins and thus represent attractive targets for the diagnosis and treatment of cancer. In complementary studies, we are collaborating with Dr. Stefanie Jeffrey (Stanford) to perform ABPP on primary human breast tumors. In addition to confirming that enzymes like KIAA1363 are legitimate new markers of breast cancer, these studies have generated enzyme activity patterns that distinguish breast tumors based on grade and clinical outcome.<p>

<center><hr width=85%></center>
<p>
<strong>Advancing the ABPP technology.</strong><br> We are pursuing the advancement of the ABPP technology to address several scientific challenges. Examples of recent innovations are summarized below.
<p>

<em>Expanding the proteome coverage of ABPP.</em> 
We and others have designed ABPP probes for several enzyme classes based on well-known covalent inhibitors [2,19]. However, cognate affinity labels do not yet exist for many enzymes. To extend ABPP to additional (and eventually all) enzyme families, we have introduced a "non-directed" version of this technology in which libraries of structurally diverse probes are synthesized and screened for their specific proteome reactivities [14]. These studies have yielded active site probes for several enzyme classes [14,20,21], none of which could be formerly targeted by ABPP. Complementing this non-directed approach, we have also developed ABPP probes for previously inaccessible enzyme families, such as the metalloproteases, by coupling broad spectrum, non-covalent inhibitors to photoreactive groups [13]. We are currently expanding our non-directed and photoreactive ABPP methods to create probes for additional enzyme classes, including kinases, phosphatases, and histone deacetylases. These innovations should greatly expand the number of enzyme classes addressable by ABPP and facilitate the systematic analysis of their function in human disease.

<p>

<em>Profiling enzyme activities in vivo.</em> 
Standard protocols for ABPP require that cells and tissues are first homogenized prior to treatment with probes, which are typically too large to permeate living samples. However, in vitro proteomic preparations may, in certain cases, fail to report accurately on the activity of enzymes in vivo, where the relative concentrations and distributions of these proteins and their regulatory partners are tightly controlled. To address this problem, we have created a tag-free version of ABPP for profiling enzyme activities in living cells and animals [6,22]. This innovation enables the functional analysis of enzymes in their native settings, thus facilitating the discovery of proteins that are selectively active in vivo.
<p>

<em> Inhibitor discovery by competitive ABPP.</em> 
ABPP provides systems-wide information on the activity state of enzymes, leading to the formulation of new hypotheses to explain the molecular basis for human disease. Testing these hypotheses, however, requires additional tools and assays. In particular, potent and selective inhibitors are crucial for the pharmacological analysis of enzyme function in vivo. We have introduced a competitive form of ABPP for inhibitor discovery that is free of the major restrictions that encumber conventional screening methods [23] (Figure 3). First, enzymes do not need to be recombinantly expressed or purified for analysis. Second, no knowledge of substrates is required, allowing inhibitors to be developed for uncharacterized enzymes. Most importantly, because inhibitors are tested against numerous enzymes in parallel, promiscuous agents can be discarded in favor of compounds that show exceptional target specificity. Using competitive ABPP, we have developed the first potent and selective inhibitors of FAAH [23], which have confirmed this enzyme's role in pain regulation [24], as well as lead inhibitors of KIAA1363 to test its function in cancer [23]. This innovation enables the rapid and systematic discovery of selective inhibitors to test the function and therapeutic potential of enzymes. These inhibitors can also serve as leads for drug design.

<p>
<center><img src="images/inhib_abpp.jpg"></center>
<p>
<em>Figure 3. Inhibitor discovery by competitive ABPP [23]. Inhibitor-enzyme interactions are measured directly in whole proteomes (examples of inhibitor-sensitive enzymes are marked with arrowheads).
</em>
<p>

<em>Substrate discovery by global metabolite profiling.</em> 
The (patho)physiological functions of enzymes can be evaluated using cell- and organism-based phenotypic screens; however, the assignment of endogenous biochemical functions to enzymes remains challenging, due in large part to a dearth of analytical methods that can be systematically applied to complex biological samples. As a result, the physiological substrates for many enzymes remain unknown. We have recently introduced a metabolomic strategy to identify the endogenous substrates of enzymes [25]. Using this discovery metabolite profiling (DMP) method (Figure 4), we have characterized a novel class of brain lipids, the taurine-conjugated fatty acids (N-acyl taurines), that are regulated by FAAH in vivo [25]. This innovation enables the systematic discovery of endogenous substrates of enzymes, thus facilitating their integration into the global metabolic and signaling networks of cells and tissues.
<p>
<center><img src="images/dmp.jpg"></center>
<p>

<em>Figure 4. Substrate discovery by global metabolite profiling. A, Comparison of conventional (targeted) metabolite analysis (Top) and discovery metabolite profiling (DMP, Bottom). B, DMP of FAAH-KO and WT brains, where relative metabolite levels are shown on a 3D plot. Both known (N-acyl ethanolamines [NAEs]) and novel (N-acyl taurines [NATs]) FAAH substrates were identified [25], while other lipids (e.g., groups 1-3) were unaltered by the deletion of FAAH.
</em>
<p>
<center><hr width=85%></center>
<p>
<a name="protomap"></a>
<strong>Technology Development: Protease Substrate Identification (<a href="/cravatt/protomap">PROTOMAP</a>)</strong><br>
The human genome encodes over 500 proteases that are involved in diverse physiological and pathological processes including tissue development, blood clotting, and cancer.  Despite the profound importance of these enzymes, many of them are wholly or partially uncharacterized with respect to their substrate repertoires.  We have developed activity based probes (ABPs) that can monitor the activities of several families of proteases including serine proteases and metalloproteases.  The identification of proteases that display altered activity in disease naturally points to their substrates as potential regulators of the disease process.  In order to identify substrates of uncharacterized proteases, we developed a comparative proteomic method called <i>Protein Topography and Migration Analysis Platform</i> (PROTOMAP).  To perform a PROTOMAP analysis, proteins are separated via 1D-SDS-PAGE and each gel-lane is cut into bands.  The bands are individually analyzed by reverse-phase LC-MS/MS and data are integrated into a unique visual-format called a <i>peptograph</i> which plots protein-sequence coverage in the horizontal dimension (N- to C-terminus, left to right) and molecular weight in the vertical dimension (high- to low-MW, top to bottom).  Peptographs are assembled for each protein in the sample thereby facilitating the visualization of shifts-in-migration and differences in protein topography that are typical of cleaved protease substrates.  This approach can be applied directly in native proteomes and can thus be used to de-orphanize proteases that have been genetically or pharmacologically perturbed.  In the past we have applied PROTOMAP to identify novel substrates of caspases during apoptosis and we are presently using PROTOMAP to identify the substrates of ucharacterized proteases that are associated with cancer pathogenicity.  More information on PROTOMAP can be found <a href="/cravatt/protomap">here</a>.
<p>
<center><img src="images/protomap_web_fig_v2.jpg"></center>
<em>Figure 5.  Protease substrate identification by proteome-wide detection of shifts in the gel-migration and topography of proteins.</em>
<p><hr width=85%>
<strong>Basic Discovery: The Enzymatic Regulation of Chemical Signaling</strong><br>
We are interested in understanding the roles that specific enzymes play in chemical signaling pathways, especially in the nervous system and cancer. As outlined above, we pursue the identification of these enzymes using functional proteomic methods such as ABPP, which have proven pivotal for our discovery of FAAH [26], an integral membrane enzyme that inactivates the fatty acid amide (FAA) family of signaling lipids [27], and KIAA1363, a novel cancer-associated hydrolase [4]. Having identified such enzymes, we use multidisciplinary research approaches to elucidate their functions in vivo. Here, I will highlight key discoveries that have emerged from our studies on the FAAH-FAA system and outline the future directions of our research program.

<p>
<em>The enzymatic regulation of FAA signaling in the nervous system.</em> Information transfer in the nervous system is carried out by chemical signals, or neurotransmitters, that traverse synapses and act on receptors to excite or inhibit target cells. Neurons utilize a range of chemical transmitters, including monoamines, neuropeptides, and lipids, which are each regulated by specific enzymes to ensure tight spatial and temporal control over their activity [28]. FAAs are a large class of signaling lipids that includes the endocannabinoid anandamide [29]
 and the sleep-inducing substance oleamide [30]. While individual FAAs act on distinct receptors in vivo, they share a common route for catabolism mediated principally by FAAH (Figure 5). Thus, tools and models to probe FAAH function are critical for understanding the FAA signaling system.
<p>
<center><img src="images/faah_cartoon.gif"></center>
<p>
<em>Figure 6. Fatty Acid Amide Hydrolase (FAAH). Cartoon model showing FAAH-catalyzed hydrolysis of anandamide to arachidonic acid, leading to the inactivation of this endocannabinoid signaling molecule.
</em>
<p>

<em>Physiological studies of FAAH.</em> We have generated FAAH-knockout (FAAH-KO) mice and shown that these animals possess greatly elevated brain levels of FAAs that correlate with enhanced cannabinoid receptor (CB1)-dependent analgesia [31]. These results indicate that pain pathways are under the influence of a FAAH-regulated endocannabinoid tone. Consistent with this notion, we and others have found that FAAH inhibitors also reduce pain through a CB1-dependent mechanism [24,32]. More recently, we have devised a transgenic method to restrict the expression of FAAH to specific tissues, allowing us to control the anatomy of FAA signaling in vivo and determine, for example, that peripheral FAAs regulate inflammation [33]. These discoveries indicate that FAAH is a primary regulator of FAA signaling in vivo and represents an attractive target for the treatment of pain and inflammatory disorders.
<p>

<em>Structural and mechanistic studies of FAAH.</em> In collaboration with Dr. Ray Stevens (TSRI), we have determined the first crystal structure of FAAH, which revealed a remarkable hydrophobic plateau domain for monotopic membrane integration [34] (Figure 6). This domain resides adjacent to FAAH's active site, suggesting that the enzyme may recruit its FAA substrates directly from the lipid bilayer. In related mechanistic studies, we have shown that FAAH utilizes a highly unusual serine-serine-lysine triad for catalysis [35,36], which distinguishes this enzyme from typical serine hydrolases that possess a serine-histidine-aspartate triad. These discoveries provide a molecular model to explain how FAAH regulates FAA signaling in vivo and offer new strategies for the design of specific inhibitors of this emerging therapeutic target.

<p>
<center><img src="images/faah.jpg"></center>
<p>
<em>Figure 7. Structure of FAAH as determined by x-ray crystallography [34]. Protein Data Bank accession number 1MT5.
</em>
<p>
<center><hr width=85%></center>
<p>


<strong>Current and future studies.</strong><br>
Our ongoing basic discovery efforts are dedicated to addressing several fundamental questions, including: Does FAAH (and, by extension, other monotopic integral membrane enzymes, e.g. cyclooxygenases [37]) recruit its lipid substrates directly from the cell membrane, and if so, how? What are the functions and sites of action of "orphan" FAAs regulated by FAAH, including the N-acyl taurines recently identified by the DMP technology? How are FAAs biosynthesized and released from cells? Does the FAAH-FAA system, as has been recently suggested [38], play a role in human cancer? Likewise, do uncharacterized hydrolases like KIAA1363 contribute to tumorigenesis and, if so, by what mechanism?
<p>

<em>Integration of Technology Development and Basic Discovery Research Programs.</em> Our ultimate goal is to create an integrated platform for the functional analysis of any mammalian enzyme. This platform should operate from the earliest stages of enzyme discovery to the final validation of therapeutic potential, allowing the biology (rather than technology) to dictate the selection of enzymes for investigation. We believe that this objective can be realized through the confluence of innovative proteomic/metabolomic technologies and focused basic discovery research, as outlined in Figure 1. The ABPP technology offers unprecedented opportunities to identify enzymes with activity profiles linked to human disease. These enzymes are then incorporated into our basic discovery research program, where we develop specific reagents to test their function (e.g., antibodies, RNAi probes, knockout mice). Concurrently, competitive ABPP screens [23] are initiated to identify selective inhibitors of these enzymes. The phenotypic impact of inactivating these enzymes (by chemical and/or genetic methods) is then evaluated using a combination of analytical chemistry, cell biology, and whole organism assays. Taken together, the output of this platform is at least three-fold: 1) novel technologies for the global analysis of enzyme function, 2) tools and models for the detailed characterization of individual enzymes, and 3) a comprehensive understanding of the role that specific enzymes play in mammalian biology. In the case of FAAH, our proteomic (ABPP) and metabolomic (DMP) technologies have led to the discovery of new inhibitors [23,24] and endogenous substrates [25] that have greatly enriched our understanding of this enzyme's functions in vivo. Moreover, the synergy achieved by integrating cutting-edge technologies with discovery research accelerates the therapeutic validation of enzymes that, like FAAH, constitute new drug targets for the burgeoning age of molecular medicine.

<p>
<center><hr width=85%></center>
<p>

<strong>References</strong><p> 

1. Kobe, B. and Kemp, B.E. Active site-directed protein regulation. Nature 1999, 402, 373-376. <p>
2. Speers, A.E. and Cravatt, B.F. Chemical strategies for activity-based proteomics. Chembiochem 2004, 5, 41-47. <a href="pdf/Speers2004.pdf">[PDF]</a><p>
3. Jessani, N. and Cravatt, B.F. The development and application of methods for activity-based protein profiling. Curr. Opin. Chem. Biol. 2004, 8, 54-59. <a href="pdf/Jessani_Cravatt2004.pdf">[PDF]</a><p> 
4. Jessani, N., Liu, Y., Humphrey, M., and Cravatt, B.F. Enzyme activity profiles of the secreted and membrane proteome that depict cancer invasiveness. Proc. Natl. Acad. Sci. U.S.A. 2002, 99, 10335-10340. <a href="pdf/Jessanietal2002.PNAS.pdf">[PDF]</a><p>
5. Jessani, N., Humphrey, M., McDonald, W.H., Niessen, S., Gangadharan, B., Yates, J.R., 3rd, Mueller, B.M., and Cravatt, B.F. Carcinoma and stromal enzyme activity profiles associated with breast tumor growth in vivo. Proc. Natl. Acad. Sci. U.S.A. 2005, in press. <p>
6. Speers, A.E. and Cravatt, B.F. Profiling enzyme activities in vivo using click chemistry methods. Chem. Biol. 2004, 11, 535-546. <a href="pdf/SpeersCravattCB2004.pdf">[PDF]</a><p>
7. Cravatt, B.F. and Lichtman, A.H. Fatty acid amide hydrolase: an emerging therapeutic target in the endocannabinoid system. Curr. Opin. Chem. Biol. 2003, 7, 469-75. <a href="pdf/CravattLichtman2003.pdf">[PDF]</a><p>
8. Patterson, S.D. and Aebersold, R. Proteomics: the first decade and beyond. Nat. Genet. 2003, 33, 311-323. <p>
9. Anderson, N.L. and Anderson, N.G. Proteome and proteomics: new technologies, new concepts, and new words. Electrophoresis 1998, 19, 1853-1861. <p>
10. Aebersold, R. and Mann, M. Mass spectrometry-based proteomics. Nature 2003, 422, 198-207. <p>
11. Liu, Y., Patricelli, M.P., and Cravatt, B.F. Activity-based protein profiling: the serine hydrolases. Proc. Natl. Acad. Sci. U.S.A. 1999, 96, 14694-14699. <a href="pdf/Liuetal1999PNAS.pdf">[PDF]</a><p>
12. Kidd, D., Liu, Y., and Cravatt, B.F. Profiling serine hydrolase activities in complex proteomes. Biochemistry 2001, 40, 6107-6115. <a href="pdf/Kiddetal2001Biochem.pdf">[PDF]</a><p>
13. Saghatelian, A., Jessani, N., Joseph, A., Humphrey, M., and Cravatt, B.F. Activity-based probes for the proteomic profiling of metalloproteases. Proc. Natl. Acad. Sci. U.S.A. 2004, 101, 10000-10005. <a href="pdf/Saghatelian2004.pdf">[PDF]</a><p>
14. Adam, G.C., Sorensen, E.J., and Cravatt B.F. Proteomic profiling of mechanistically distinct enzyme classes using a common chemotype. Nat. Biotechnol. 2002, 20, 805-809. <a href="pdf/Adametal2002NatBiotech.pdf">[PDF]</a><p>
15. Adam, G.C., Burbaum, J.J., Kozarich, J.W., Patricelli, M.P., and Cravatt, B.F. Mapping enzyme active sites in complex proteomes. J. Amer. Chem. Soc. 2004, 126, 1363-1368. <a href="pdf/Adam2004.pdf">[PDF]</a><p>
16. Ince, T.A. and Weinberg, R.A. Functional genomics and the breast cancer problem. Cancer Cell 2002, 1, 15-7. <p>
17. Sorlie, T., et al. Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. Proc. Natl. Acad. Sci. U.S.A. 2001, 98, 10869-74. <p>
18. van 't Veer, L.J., et al. Gene expression profiling predicts clinical outcome of breast cancer. Nature 2002, 415, 530-6. <p>
19. Jeffery, D.A. and Bogyo, M. Chemical proteomics and its application to drug discovery. Curr. Opin. Biotechnol. 2003, 14, 87-95. <p>
20. Adam, G.C., Sorensen, E.J., and Cravatt, B.F. Trifunctional chemical probes for the consolidated detection and identification of enzyme activities from complex proteomes. Mol. Cell. Proteomics 2002, 1, 828-835. <a href="pdf/AdametalMCP2002.pdf">[PDF]</a><p>
21. Barglow, C.T., Cravatt, B.F. Discovering disease-associated enzymes by proteome reactivity profiling. Chem. Biol. 2004, 11, 1523-1531. <a href="pdf/Barglow2004.pdf">[PDF]</a><p>
22. Speers, A.E., Adam, G.C., and Cravatt, B.F. Activity-based protein profiling in vivo using a copper(I)-catalyzed azide-alkyne [3 + 2] cycloaddition. J. Amer. Chem. Soc. 2003, 125, 4686-4687. <a href="pdf/Speersetal2003.pdf">[PDF]</a><p>
23. Leung, D., Hardouin, C., Boger, D.L., and Cravatt, B.F. Discovering potent and selective reversible inhibitors of enzymes in complex proteomes. Nat. Biotechnol. 2003, 21, 687-691. <a href="pdf/Leungetal2003.pdf">[PDF]</a><p>
24. Lichtman, A.H., Leung, D., Shelton, C., Saghatelian, A., Hardouin, C., Boger, D., and Cravatt, B.F. Reversible inhibitors of fatty acid amide hydrolase that promote analgesia: evidence for an unprecedented combination of potency and selectivity. J. Pharmacol. Exp. Ther. 2004, 311, 441-448. <a href="pdf/Lichtman2004b.pdf">[PDF]</a><p>
25. Saghatelian, A., Trauger, S.A., Want, E.J., Hawkins, E.G., Siuzdak, G., and Cravatt, B.F. Assignment of endogenous substrates to enzymes by global metabolite profiling. 2004, 43, 14332-14339. <a href="pdf/SaghatelianTrauger2004.pdf">[PDF]</a><p>
26. Cravatt, B.F., Giang, D.K., Mayfield, S.P., Boger, D.L., Lerner, R.A., and Gilula, N.B. Molecular characterization of an enzyme that degrades neuromodulatory fatty-acid amides. Nature 1996, 384, 83-87. <a href="pdf/Cravatt1996.pdf">[PDF]</a><p>
27. Patricelli, M.P. and Cravatt, B.F. Proteins regulating the biosynthesis and inactivation of neuromodulatory fatty acid amides. Vitam. Horm. 2001, 62, 95-131. <p>
28. Siegel, G.J., Agranoff, B.W., Albers, R.W., and Molinoff, P.B., Basic Neurochemistry 1994, New York: Raven Press. 182-183. <p>
29. Devane, W.A., et al. Isolation and structure of a brain constituent that binds to the cannabinoid receptor. Science 1992, 258, 1946-9. <p>
30. Cravatt, B.F., Prospero-Garcia, O., Siuzdak, G., Gilula, N.B., Henriksen, S.J., Boger, D.L., and Lerner, R.A. Chemical characterization of a family of brain lipids that induce sleep. Science 1995, 268, 1506-1509. <p>
31. Cravatt, B.F., Demarest, K., Patricelli, M.P., Bracey, M.H., Giang, D.K., Martin, B.R., and Lichtman, A.H. Supersensitivity to anandamide and enhanced endogenous cannabinoid signaling in mice lacking fatty acid amide hydrolase. Proc. Natl. Acad. Sci. U.S.A. 2001, 98, 9371-9376. <a href="pdf/CravattetalPNAS2001.pdf">[PDF]</a><p>
32. Kathuria, S., et al. Modulation of anxiety through blockade of anandamide hydrolysis. Nat. Med. 2003, 9, 76-81. <p>
33. Cravatt, B.F., Saghatelian, A., Hawkins, E.G., Clement, A.B., Bracey, M.H., and Lichtman, A.H. Functional disassociation of the central and peripheral fatty acid amide signaling systems. Proc. Natl. Acad. Sci. U.S.A. 2004, 101, 10821-10826. <a href="pdf/CravattSaghatelian2004.pdf">[PDF]</a><p>
34. Bracey, M.H., Hanson, M.A., Masuda, K.R., Stevens, R.C., and Cravatt, B.F. Structural adaptations in a membrane enzyme that terminates endocannabinoid signaling. Science 2002, 298, 1793-1796. <a href="pdf/Bracey2002.pdf">[PDF]</a><p>
35. McKinney, M.K. and Cravatt, B.F. Evidence for distinct roles in catalysis for residues of the serine-serine-lysine catalytic triad of fatty acid amide hydrolase. J. Biol. Chem. 2003, 278, 37393-37399. <a href="pdf/mckinney2003.pdf">[PDF]</a>
<p>
36. Patricelli, M.P. and Cravatt, B.F. Fatty acid amide hydrolase competitively degrades bioactive amides and esters through a non-conventional catalytic mechanism. Biochemistry 1999, 38, 14125-14130. <a href="pdf/PatricelliCravatt1999.pdf">[PDF]</a><p>
37. Picot, D., Loll, P.J., and Garavito, R.M. The X-ray crystal structure of the membrane protein prostaglandin H2 synthase-1. Nature 1994, 367, 243-249. <p>
38. Bifulco. M., Laezza, C., Valenti, M., Ligresti, A., Portella, G., Di Marzo, V. A new strategy to block tumor growth by inhibiting endocannabinoid inactivation. FASEB J. 2004, 18, 1606-1608. 
<p>








</td>
<td width=20%></td>
</tr>
</table>
<br><p><br>